core/src/test/java/io/github/tcdl/messages/MessageFactoryTest.java

```diff
@@ -131,8 +134,7 @@ public void testCreateMessageBuilderMetaFromMsgOptions() throws Exception {
 
         Message message = mesageBuilder.build();
         assertEquals("Message ttl is not set correctly", ttl, message.getMeta().getTtl());
-        assertEquals("Message create_at is not equals now", DateUtils.truncate(new Date(), Calendar.SECOND),
-                DateUtils.truncate(message.getMeta().getCreatedAt(), Calendar.SECOND));
+        assertEquals("Message create_at is not equals now", FIXED_CLOCK_INSTANT, message.getMeta().getCreatedAt());
```

Comment:
This is a place were the new approach really shines. Instead of non-reliable truncating that we had before we can check for exact equality.